ted_sws/data_manager/services/export_notice_from_mongodb.py

```diff
@@ -0,0 +1,72 @@
+import base64
+import json
+import pathlib
+import zipfile
+from typing import Union
+
+from pymongo import MongoClient
+
+from ted_sws import config
+from ted_sws.core.model.notice import Notice
+from ted_sws.data_manager.adapters.notice_repository import NoticeRepository
+
+
+def save_notice_as_zip(notice: Notice, unpack_path: pathlib.Path):
+    """
+    Serialize notice and saves its information as a .zip file.
+    :param notice:
+    :param unpack_path:
+    """
+
+    def write_in_file(data: Union[str, bytes], terminal_path: str):
+        write_path = unpack_path / terminal_path
+        if type(data) == str:
+            write_path.write_text(data=data, encoding="utf-8")
+        elif type(data) == bytes:
+            write_path.write_bytes(data)
+
+    write_in_file(notice.rdf_manifestation.object_data, "rdf_manifestation.ttl")
+    write_in_file(notice.distilled_rdf_manifestation.object_data, "distilled_rdf_manifestation.ttl")
```

Comment:
In general case distilled_rdf_manifestation can be None and here code will fail.